packages/fluent-vue/src/vue/component.ts

```diff
@@ -40,6 +41,22 @@ export default defineComponent({
       return fluent.format(props.path, params)
     })
 
+    const translationAttrs = computed(() => {
+      if (!props.useTa) {
+        return null
+      }
+
+      const params = Object.assign(
+        {},
+        props.args,
+        ...Object.keys(childSlots).map((key) => ({
+          [key]: `\uFFFF\uFFFE${key}\uFFFF`,
+        }))
+      )
+
+      return fluent.formatAttrs(props.path, params)
```

Comment:
I'm thinking `translationAttrs` here should be converted to `camelCase`, to prevent manually changing it on destructure. As fluent default key format is `kebab-case` and it is `camelCase` in Vue.
This probably should be the default for `formatAttrs` method too.
What do you think?

Reply:
I was thinking the same thing. Vue itself already does this king of case conversion, so it's not unusual. That change to formatAttrs would be a breaking one though, right?

Reply:
I'll change formatAttrs and check if all tests still pass

Reply:
It will be breaking change only for those that use `formatAttrs` directly.
`$ta` method is meant to be used to bind component props and Vue will convert casing anyway, so that should be fine.
As version 3 of this library is still in beta I'm fine with breaking things a little.

Reply:
I've experimented changing the default of formatAttrs and all tests passed, but only because there's no test using kebab attributes. I then added this test:

```ts
it('renders fluent kebab attributes correctly', () => {
    // Arrange
    bundle.addResource(
      new FluentResource(ftl`
      key =
        .kebab-attr = Attr value
      `)
    )

    const component = {
      template: `<div v-bind="$ta('key')"></div>`,
    }

    // Act
    const mounted = mount(component, options)

    // Assert
    expect(mounted.html()).toEqual(`<div kebab-attr="Attr value"></div>`)
  })
```

And changed this bit from formatAttrs:
```ts
    for (const [attrName, attrValue] of Object.entries(message.attributes)) {
      result[camelize(attrName)] = this.formatPattern(context, attrValue, value)
    }
```

But this was the test result:

```
  ● method › renders fluent kebab attributes correctly

    expect(received).toEqual(expected) // deep equality

    Expected: "<div kebab-attr=\"Attr value\"></div>"
    Received: "<div kebabattr=\"Attr value\"></div>"
```

Reply:
TIL: Vue does not convert casing for arbitrary attributes. Then we keep conversion just in component.